routes/api.routes.js

```diff
@@ -0,0 +1,13 @@
+const express = require('express');
+const router = express.Router();
+const apiController = require('../controllers/api.controller');
```

Comment:
Api routes are now truly what they are meant to be: endpoints with no logic that call for certain functions.